src/puppetlabs/puppetdb/command.clj

```diff
@@ -210,8 +216,8 @@
     (jdbc/with-transacted-connection db
       (scf-storage/maybe-activate-node! certname producer-timestamp)
       (scf-storage/add-report! report received))
-    (log/infof "[%s] [%s] puppet v%s - %s"
-               id
+    (log/infof "[%s-%d] '%s' puppet v%s command processed for %s"
+               id, (tcoerce/to-long received)
```

Comment:
I typically use a comma when some lines have only one argument and other lines have multiple, but I'm not too attached to it. I'll add the `trs`